hbook/ntup/ntup_example_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ntup
```

Comment:
please put examples into `_test` packages so the example code shows the exported API (and doesn't use internal bits.)
_i.e.:_
`s/package ntup/package ntup_test/`